packages/dhis-api/src/buildAnalyticsQuery.js

```diff
@@ -75,6 +75,26 @@ export const buildDataValueAnalyticsQueries = queryInput => {
   return queries;
 };
 
+export const buildAggregatedDataValueAnalyticsQueries = queryInput => {
+  const newQueryInput = { ...queryInput };
+
```

Comment:
Suggest comment:

All data in dhis has a date granularity, e.g. you can submit an event against a full date, or just December 2018. 

When we are fetching raw data, we can ignore this, because if we request data for 2015-2021 it will return all data, including an event at December 2018 because it is within that range.

But when we fetch aggregated data, the API behaves differently. If we request data for 2015-2021 it will NOT return an event at December 2018. To get the event, we have to request data for 201810 specifically.

This is true for both fetching data element values and fetching indicator values, however, we use the raw data API endpoint most of the time, and we only use the aggregated API endpoint for indicators. So practically, this is an indicator problem, but technically it's for both.